apps/bitwarden-mailconfig.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+# Check if Bitwarden is already installed
+print_text_in_color "$ICyan" "Checking if Bitwarden is already installed..."
+if is_docker_running
+then
+    if docker ps -a --format '{{.Names}}' | grep -Eq "bitwarden";
+    then
+        if [ ! -d "$BITWARDEN_HOME"/bwdata ]
+        then
+            msg_box "It seems like 'Bitwarden' isn't installed in $BITWARDEN_HOME.\n\nYou cannot run this script."
+            exit 1
+        fi
+    else
+        msg_box "It seems like 'Bitwarden' isn't installed.\n\nYou cannot run this script."
+        exit 1
+    fi
+else
+    msg_box "It seems like 'Bitwarden' isn't installed.\n\nYou cannot run this script."
+    exit 1
+fi
+
+msg_box "This script lets you configure your mailserver settings for Bitwarden."
+if [[ "no" == $(ask_yes_or_no "Do you want to continue?") ]]
+then
+    exit
+fi
+
+# Insert globalSettings__mail__smtp__trustServer to global.override
+if ! grep -q "^globalSettings__mail__smtp__trustServer=" "$BITWARDEN_HOME"/bwdata/env/global.override.env
+then
+    echo "globalSettings__mail__smtp__trustServer=false" >> "$BITWARDEN_HOME"/bwdata/env/global.override.env
+fi
+
+# Insert globalSettings__mail__smtp__startTls to global.override
+if ! grep -q "^globalSettings__mail__smtp__startTls=" "$BITWARDEN_HOME"/bwdata/env/global.override.env
+then
+    echo "globalSettings__mail__smtp__startTls=false" >> "$BITWARDEN_HOME"/bwdata/env/global.override.env
+fi
+
+# Enter mailserver
+while true
+do
+    MAIL_SERVER=$(whiptail --inputbox "Please enter the mailserver URL that you want to use.\nE.g. smtp.mail.de\nIf you don't want to change the mailserver, that is already configured inside the global.override.env-file, just leave the box empty." "$WT_HEIGHT" "$WT_WIDTH" 3>&1 1>&2 2>&3)
+    if [[ "no" == $(ask_yes_or_no "Is this correct? $MAIL_SERVER") ]]
+    then
+        print_text_in_color "$ICyan" "OK, please try again."
```

Comment:
btw: if you don't want a msg_box here, I would suggest to add a `sleep 2` after this, else you won't be able to read it.

Reply:
This of course is meant for all places where you have done that.

Reply:
`sleep 1` is probably enough